mailbox/api/src/main/java/org/apache/james/mailbox/MailboxManager.java

```diff
@@ -130,6 +130,21 @@
     MessageManager getMailbox(MailboxPath mailboxPath, MailboxSession session) throws MailboxException;
 
     /**
+     * Gets an session suitable for IMAP.
```

Comment:
Copy paste detected;)
I'm not found of this sentance, may be something like: `Gets an object managing the given mailbox`